Lib/fontTools/ttLib/ttFont.py

```diff
@@ -260,7 +260,7 @@ def _save(self, file, tableCache=None):
             ]  # make sure 'head' is loaded so the recalculation is actually done
 
         tags = list(self.keys())
-        if "GlyphOrder" in tags:
+        if "GlyphOrder" in self.keys():
```

Comment:
TTFont.keys() does a bunch of things that you don't need here to check if a tag is present, please revert to the old approach here and elsewhere you used this patterm